doc/boltek_LD-250_sensor.rst

```diff
@@ -0,0 +1,105 @@
+=================================================================
+Serial protocol definition for: Boltek LD-250 Lightning Detector.
+=================================================================
+
+Wouter van Reeven.
+2022-11-11
+
+Serial Interface & Parameters
+=============================
+
+| Serial Interface: RS-232
+| BAUD: 19200
+| Bits: 8
+| Parity: None
+| Stop Bits: 1
+| Flow Control: None
+| Data type: ISO8859-1
+
+Data output by the LD-250 instrument can be status data:
+
+    - Status data indicator
+    - Close strike rate (strikes/minute)
+    - Total strike rate (strikes/minute)
+    - Close alarm status (0: not active, 1: active)
+    - Severe alarm status (0: not active, 1: active)
+    - Sensor azimuth heading (deg)
+
+or noise data
+
+    - Noise data indicator
+
+or strike data
+
+    - Strike data indicator
+    - Corrected strike distance (miles)
+    - Uncorrected strike distance (miles)
+    - Strike azimuth bearing (deg)
+
+Status data is output as it is calculated by the instrument at a fixed period of approximately 1 s.
+Noise data is output if the noise level in the sensor is too high and the strike signal is lost.
+The level, below which noise is squelched, can be set in the configuration of the instrument.
+Strike data is output as it is read by the instrument and depends on when a strike is detected.
+A line terminator is added to the serial data line.
+
+Example Output Lines
+====================
+
+Status data:
+
+| '$WIMST,0,0,0,0,000.0*42\\r\\n'
+
+Noise data:
+
+| '$WIMLN*CE\\r\\n'
+
+Strike data:
+
+| '$WIMLI,10,11,000.0*42\\r\\n'
+
+Line Formats
+============
+
+A status data line consists of:
+
+    - $-sign to indicate the start of the telemetry.
+    - Status data indicator. This is always 'WIMST'.
+    - Close strike rate (strikes/minute). Integer value between 0 and 999.
+    - Total strike rate (strikes/minute). Integer value between 0 and 999.
+    - Close alarm status (0: not active, 1: active)
+    - Severe alarm status (0: not active, 1: active)
+    - Sensor azimuth heading (deg). Zero padded decimal value between 000.0 and 359.9.
```

Comment:
Please document the sign convention (i.e. if 0 is north, is 90 east or west) and say whether that matches our observatory standard.

That could go here, or when azimuth is first mentioned above, and some sort of reference might help below for the strike format.